mix compile failed on dashing-ex1.11.4-otp23.3.1

https://github.com/rclex/rclex/runs/2711160555?check_suite_focus=true

```
source /opt/ros/dashing/setup.bash
  source /opt/ros/dashing/setup.bash
  cd rclex
  mix local.hex --force
  mix deps.get
  mix compile
  shell: bash --noprofile --norc -e -o pipefail {0}

13:37:15.504 [warn]  Description: 'Authenticity is not established by certificate path validation'
     Reason: 'Option {verify, verify_peer} and cacertfile/cacerts is missing'


13:37:15.651 [warn]  Description: 'Authenticity is not established by certificate path validation'
     Reason: 'Option {verify, verify_peer} and cacertfile/cacerts is missing'


13:37:15.708 [warn]  Description: 'Authenticity is not established by certificate path validation'
     Reason: 'Option {verify, verify_peer} and cacertfile/cacerts is missing'

* creating /github/home/.mix/archives/hex-0.21.2
Resolving Hex dependencies...
Dependency resolution completed:
Unchanged:
  earmark_parser 1.4.9
  elixir_make 0.6.2
  ex_doc 0.24.2
  makeup 1.0.3
  makeup_elixir 0.14.1
  makeup_erlang 0.1.1
  nimble_parsec 0.6.0
* Getting elixir_make (Hex package)
* Getting ex_doc (Hex package)
* Getting earmark_parser (Hex package)
* Getting makeup_elixir (Hex package)
* Getting makeup_erlang (Hex package)
* Getting makeup (Hex package)
* Getting nimble_parsec (Hex package)
==> earmark_parser
Compiling 1 file (.yrl)
could not compile dependency :earmark_parser, "mix compile" failed. You can recompile this dependency with "mix deps.compile earmark_parser", update it with "mix deps.update earmark_parser" or clean it with "mix deps.clean earmark_parser"
** (Mix) Could not compile "src/link_text_parser.yrl" because the application "parsetools" could not be found. This may happen if your package manager broke Erlang into multiple packages and may be fixed by installing the missing "erlang-dev" and "erlang-parsetools" packages
==> rclex
Error: Process completed with exit code 1.
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -12,7 +12,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        dockertags: [latest, dashing-ex1.12.0-otp24.0.1, dashing-ex1.11.4-otp23.3.1, dashing-ex1.10.4-otp22.3.4, dashing-ex1.9.4-otp22.3.4, dashing-ex1.9.1-otp22.0.7]
+        dockertags: [latest, dashing-ex1.12.0-otp24.0.1, dashing-ex1.11.2-otp23.3.1, dashing-ex1.10.4-otp22.3.4, dashing-ex1.9.4-otp22.3.4, dashing-ex1.9.1-otp22.0.7]
     container: rclex/rclex_docker:${{ matrix.dockertags }}
 
     steps:

